html/semantics/popovers/popover-attribute-all-elements.html

```diff
@@ -33,7 +33,23 @@
       assertNotAPopover(element);
     }, `A <${tag}> element should *not* behave as a popover.`);
   });
-
+  elementsThatDontRender.forEach(tag => {
+    test((t) => {
+      const element = document.createElement(tag);
+      element.setAttribute('popover','auto');
+      document.body.appendChild(element);
+      t.add_cleanup(() => element.remove());
+      assertPopoverVisibility(element, false, false, 'should not behave as a popover');
```

Comment:
Being non-rendered content shouldn't affect whether the popover attribute applies, or whether `:open`/`:closed` apply (you can perfectly do `:open { display: none }` or `:closed { display: block }`, so I don't think we should restrict this on an element-basis).

I think we shouldn't use `assertPopoverVisibility` and just assert the display:
`assert_equals(window.getComputedStyle(popover).display, "none", "Element isn't rendered");`.

If we really want to restrict the popover attribute on an element-basis, there should be a well defined list in the HTML spec.

Reply:
(Besides you made me found a bug in `assertPopoverVisibility`)

Reply:
I added the display check.